pkg/alerts/nrql_conditions.go

```diff
@@ -273,6 +275,7 @@ type NrqlConditionUpdateBase struct {
 	ViolationTimeLimitSeconds int                              `json:"violationTimeLimitSeconds,omitempty"`
 	Expiration                *AlertsNrqlConditionExpiration   `json:"expiration,omitempty"`
 	Signal                    *AlertsNrqlConditionUpdateSignal `json:"signal"`
+	IncidentTitleTemplate     *string                          `json:"incidentTitleTemplate,omitempty"`
```

Comment:
@baNROne I think we'll want to leave off `omitempty` for `NrqlConditionUpdateBase`. The reason is because we want a user to be able to remove an incident title template on an existing condition that already has the field configured. This presentation (slides 20-25 particularly) does a great job at explaining the reasoning. https://docs.google.com/presentation/d/1VEYfI7acg8LH_uGHjj5Su4RoF1lQRPVyfxK59xUMxV8/edit?usp=sharing

Reply:
Makes sense. Thanks @akane0915. I've updated it.